source/developers/cook-books/how-tos/web-app-manifest-use-credentials.rst

```diff
@@ -0,0 +1,57 @@
+:is-up-to-date: True
+
+.. index:: Enable Use of Cookies and Credentials by the Web App Manifest, Web App Manifest, Manifest
+
+===================================================
+Enabling Use of Credentials by the Web App Manifest
+===================================================
+
+Some sites uses a web app manifest, a file that describes the site.
```

Comment:
Odd sentence construction (uses => use).

Also, instead of sites, "some web apps" would be more accurate.